tests/UI/campaigns/functional/BO/08_design/06_imageSettings/13_checkProductImageFormat.ts

```diff
@@ -129,19 +124,22 @@ describe('BO - Design - Image Settings - Check product image format', async () =
     {
       product: productDataPNG,
       extOriginal: 'png',
-      extGenerated: 'png',
+      extGenerated: 'jpg',
+      extImageType: 'png',
     },
     {
       product: productDataJPG,
       extOriginal: 'jpg',
       extGenerated: 'jpg',
+      extImageType: 'jpg',
     },
     {
       product: productDataWEBP,
       extOriginal: 'webp',
       extGenerated: 'jpg',
+      extImageType: 'png',
```

Comment:
```suggestion
      extImageType: 'webp',
```

It should be `webp` and not `png`.

Reply:
No, it should be png ;) Only jpg or png for the original image.

Reply:
No, the original file must keep in the original format and must not be changed.

Reply:
The configuration impact generated files.

Reply:
@Progi1984 This is not true